tests/unit/providers/agents/meta_reference/test_response_tool_context.py

```diff
@@ -5,9 +5,24 @@
 # the root directory of this source tree.
 
 
+from llama_stack.providers.inline.agents.meta_reference.responses.streaming import (
+    _process_tool_choice,
+)
 from llama_stack.providers.inline.agents.meta_reference.responses.types import ToolContext
+from llama_stack_api import (
+    OpenAIChatCompletionToolChoiceAllowedTools,
+    OpenAIChatCompletionToolChoiceCustomTool,
+    OpenAIChatCompletionToolChoiceFunctionTool,
+)
 from llama_stack_api.openai_responses import (
     MCPListToolsTool,
```

Comment:
not sure if this is a residual from my refactor, but can't these two `from llama_stack_api` imports be combined?